PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,32 @@
+# Description
+
+Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.
```

Comment:
This is a good starting point but cleanup the template so there is only a small checklist (3-5 items). Otherwise, I have seen that if it is too many things on the template then developers ignore the entire thing.